gooddata-java/src/main/java/com/gooddata/sdk/service/account/AccountService.java

```diff
@@ -182,4 +183,15 @@ public SeparatorSettings getSeparatorSettings(final Account account) {
             throw new GoodDataException("Unable to get separators for account=" + account.getUri(), e);
         }
     }
+
+    public Projects getProjectsForUser(final Account account) {
+        notNull(account, "account");
+        notEmpty(account.getId(), "account.uri");
+
+        try {
+            return restTemplate.getForObject(ACCOUNT_TEMPLATE.expand(account.getId()) + "/projects", Projects.class);
```

Comment:
Please @liry or @peter-plochan lets agree on API - 
1) is it ok to have there Account? (we will have account loaded and it would align with previous call we will use for deprovisioning)
2) response Projects (I hope json jackson mapping will work)
3) building URL, is it ok to reuse ACCOUNT_TEMPLATE + "/projects ?

Reply:
FYI: we have listing of projects for current user implemented already in `ProjectService.listProjects` and I would add new method right next to it (and refactor them to have only one implementation). Add e.g. `PageBrowser<Project> listProjects(final Account account)`
* using `Account` in this API is perfectly fine (e.g. `User getUser(final Project project, final Account account)` uses it as well)
* I would return `PageBrowser<Project>` (see `listProjects`)
* There's already `ProjectService.LIST_PROJECTS_TEMPLATE` created

Reply:
I dont know what you mean by " they have only one implementation". I cannot change API, thus all public method should stay, I will add new public method and those may reuse private, which I will refactor if possible, ok?

Reply:
I just meant that work with the `restTemplate` should stay on one place (in `private listProjects(URI uri)`), so that's exactly what you're describing. It should be even doable w/o that private method change, just something like:
```java
    public PageBrowser<Project> listProjects(final Account account) {
        return new PageBrowser<>(page -> listProjects(getProjectsUri(account.getId(), page)));
    }
```